src/comm/TcpAcceptor.h

```diff
@@ -81,6 +74,16 @@ class TcpAcceptor : public AsyncJob
     /// if not the accept() will be postponed
     static bool okToAccept();
 
+    /** Try and accept another connection (synchronous).
+     *
+     * If one is pending already the subscribed callback handler will be
+     * scheduled to handle it before this method returns.
+     *
+     * If there are not enough FD available to do the accept(2) safely will
+     * defer the operation until later when enough sockets become available.
```

Comment:
I have pruned the detail above this one. If you think you can re-write this sentence better, suggestions are welcome.

Reply:
I suggest the following complete method description:

```C++
/// Accept a new connection now (if possible) or defer (otherwise).
/// This read handler for the listening socket may also be called directly.
```

This description does not document okToAccept() logic, addressing my original concern, and documents the dual-purpose of this method so that we do not have to re-document read handler API here. I [still](https://github.com/squid-cache/squid/pull/870#discussion_r748624529) suspect that the dual purpose is a design mistake, and the method should be split into two methods instead, but this change request is not the right place for that discussion.